### ISSUE TITLE
Update dependencies for Drupal 10

### ISSUE THREAD

**Issue by ultimike** [joachim-n/drupal-core-development-project#18]
I believe composer/installers and symfony/var-dumper need to have their version requirements bumped to support Drupal 10:

```
"composer/installers": "^2.0",
"symfony/var-dumper": "^6.2"
```

-mike

**Comment by joachim-n** [comment]
Will this need a new branch, or are those versions compatible with D9?

**Comment by ultimike** [comment]
Without testing, I suspect the `composer/installers` bump is valid for Drupal 9 and 10, but the `symfony/var-dumper` bump is only for Drupal 10 (so, a new branch for Drupal 10 I suppose).

-mike

**Comment by joachim-n** [comment]
Fixed in fd0eba4c931266003a446a6317ef5c42f030865a. 

Looks like we can support both D9 and 10 with Commposer ORed requirements -- I've tested by switching my drupal clone between 9.5 and 10.1 and doing `composer update`.

Tagging a new release.
